namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-workload-preprod/resources/reductions-completed-sqs.tf

```diff
@@ -0,0 +1,69 @@
+module "hmpps_reductions_completed_queue" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-sqs?ref=4.8"
+
+  environment-name          = var.environment
+  team_name                 = var.team_name
+  infrastructure-support    = var.infrastructure_support
+  application               = var.application
+  sqs_name                  = "hmpps_reductions_completed_event_queue"
+  encrypt_sqs_kms           = "true"
+  message_retention_seconds = 1209600
+  namespace                 = var.namespace
+
+  redrive_policy = <<EOF
+  {
+    "deadLetterTargetArn": "${module.hmpps_reductions_completed_dead_letter_queue.sqs_arn}","maxReceiveCount": 3
+  }
+  EOF
+
+  providers = {
+    aws = aws.london
+  }
+}
+
+module "hmpps_reductions_completed_dead_letter_queue" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-sqs?ref=4.8"
+
+  environment-name       = var.environment
+  team_name              = var.team_name
+  infrastructure-support = var.infrastructure_support
+  application            = var.application
+  sqs_name               = "hmpps_reductions_completed_event_dlq"
+  encrypt_sqs_kms        = "true"
+  namespace              = var.namespace
+
+  providers = {
+    aws = aws.london
+  }
+}
+
+
+resource "kubernetes_secret" "hmpps_reductions_completed_event_queue" {
+  metadata {
+    name      = "hmpps-reductions-completed-sqs-output"
```

Comment:
might better to rename to 
hmpps-reductions-completed-sqs-instance-output
to match the other queue secrets